deploy.go

```diff
@@ -313,7 +301,7 @@ func (d *imageDeployer) newEFIEnvironment() (secboot_efi.HostEnvironment, error)
 		return efienv.NewEnvironment(&config, tcglog.AlgorithmIdList{tpm2.HashAlgorithmSHA256}), nil
 	}
 
-	return nil, nil
+	return nil, fmt.Errorf("one of az disk profile or UEFI config must be available")
```

Comment:
How come this returns an error here now? The previous `(nil, nil)` return allowed a profile to be generated from the current host environment. I'm not sure if anyone is using that though.